config/locales/en.yml

```diff
@@ -219,10 +219,12 @@ en:
     event_summary: 'Refill for %{name}.'
     form:
       strength: Strength
-      strength_placeholder: e.g. 100
-      total_placeholder: e.g. 30
+      strength_unit: 'Strength Unit'
+      strength_placeholder: 'e.g. 100'
+      total_placeholder: 'e.g. 30'
```

Comment:
I don't think we need all these new placeholders. These fields must be numeric and users would be filling in this information from their prescription. But this was a good suggestion! :)

Reply:
I have removed these placeholders on my latest commit